deployment/build-and-stage.yaml

```diff
@@ -27,23 +27,23 @@ steps:
 - name: gcr.io/cloud-builders/gcloud
   entrypoint: bash
   args:
-    - "-c"
-    - |
-      read current_build _ <<< $(gcloud builds list --filter=tags="build-and-stage" --format="value(id,create_time)" --sort-by=create_time --ongoing)
+  - '-c'
+  - |
+    read current_build _ <<< $(gcloud builds list --filter=tags="build-and-stage" --format="value(id,create_time)" --sort-by=create_time --ongoing)
```

Comment:
You could remove `create_time` from the format and then you wouldn't need to have the awkward looking underscore as part of the `read`, right?

Reply:
I had tried that, but it didn't seem return the results in the correct order. My guess was that the `--sort-by` flag needs the in the output in order for it to sort.

Reply:
Really? Strange:

```
$ gcloud --project oss-vdb builds list --filter=tags="build-and-stage" --format="value(id,create_time)" --sort-by=create_time
a6f9014c-c44e-4c15-9db3-2078bbf9008d    2023-02-16T06:21:17.870948332Z
a37d3ca6-7d8f-41c4-8f70-d3bb1bce4ecd    2023-02-16T23:23:14.650251760Z
47c260a7-6013-4318-a7e6-98bd06dffff5    2023-02-16T23:27:19.943816952Z
1465d5df-3d0d-457b-8868-b064ff4b1145    2023-02-21T22:31:40.402139Z
fc854b17-04f3-452a-8c37-036871559d2e    2023-02-21T23:34:34.953340Z
18c08d02-42dd-4344-9727-d780482aa1a2    2023-02-22T03:40:17.788380Z
4289cbdc-7e84-4cb5-9e6e-2684e67ee1b4    2023-02-23T00:01:52.660899Z
7774ca68-ac28-484d-a388-5c1459893835    2023-02-23T00:55:19.186404Z
f47161bd-b62c-45f9-b8a4-8b6542899423    2023-02-23T01:06:22.864975Z
38a71182-f22c-404c-80c0-7ed5afd77ab6    2023-02-23T04:23:11.050005Z
```

```
$ gcloud --project oss-vdb builds list --filter=tags="build-and-stage" --format="value(id)" --sort-by=create_time
a6f9014c-c44e-4c15-9db3-2078bbf9008d
a37d3ca6-7d8f-41c4-8f70-d3bb1bce4ecd
47c260a7-6013-4318-a7e6-98bd06dffff5
1465d5df-3d0d-457b-8868-b064ff4b1145
fc854b17-04f3-452a-8c37-036871559d2e
18c08d02-42dd-4344-9727-d780482aa1a2
4289cbdc-7e84-4cb5-9e6e-2684e67ee1b4
7774ca68-ac28-484d-a388-5c1459893835
f47161bd-b62c-45f9-b8a4-8b6542899423
38a71182-f22c-404c-80c0-7ed5afd77ab6
```

Reply:
Hm I've just double checked this a few times in Cloud Build and it seems to be working fine.
Not sure how I was getting a wrong result before. Will remove `create_time`.